gateway/src/apicast/linked_list.lua

```diff
@@ -10,7 +10,7 @@ local noop = function() end
 
 local empty_t = setmetatable({}, { __newindex = noop })
 local __index = function(t,k)
-    return t.current[k] or t.next[k]
+    if t.current[k] == nil then return t.next[k] else return t.current[k] end
```

Comment:
Did you find any bugs caused by this?

Reply:
Yep. I could not override the `timer_resolution` with `false`. So I bundled it into one PR.